lib/grp.gi

```diff
@@ -433,11 +439,6 @@ InstallMethod( IsNilpotentGroup,
 ##
 #M  IsPerfectGroup( <G> ) . . . . . . . . . . . .  test if a group is perfect
 ##
-InstallImmediateMethod( IsPerfectGroup,
-    IsSolvableGroup and HasIsTrivial,
-    0,
-    IsTrivial );
-
```

Comment:
Why remove this, instead of turning this into a plain method? Wouldn't that still be useful?

Reply:
(I don't think it's necessary to add a "this used to be an immediate method ..." comment)